modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java

```diff
@@ -1720,6 +1716,36 @@ private void _testGetStructuredContentWithAllTypesOfContentFields(
 		assertValid(getStructuredContent);
 	}
 
+	private void _testGetStructuredContentWithDateExpired() throws Exception {
+		StructuredContent structuredContent = randomStructuredContent();
+
+		Date dateExpired = _randomDatePlusAYear();
+
+		structuredContent.setDateExpired(dateExpired);
+
+		boolean neverExpire = RandomTestUtil.randomBoolean();
```

Comment:
@gergelyszalay, can we add two tests? One for neverExpire and one for expire? Same for patch

thanks!

Reply:
Hi @jkappler separated the 2 cases with parameters in the method, is it better if I completely separate those?

Reply:
yes @gergelyszalay. it would be better, we can talk about it on Monday :)